frontend/src/index.js

```diff
@@ -10,6 +10,9 @@ import { Provider } from "react-redux";
 import reducer from "./reducer/reducer";
 import "./reset.css";
 
+const vh = window.innerHeight * 0.01;
```

Comment:
Why is that ?

Reply:
This is to get the real height of each device (I realised that the "vh" measure wasn't as precise as I thought) so I needed to do that little trick.